README.md

```diff
@@ -186,3 +186,27 @@ require setup in your environment:
   podman has better performance than rootless, so `CONTAINER_ENGINE="sudo podman"`).
 - Have mock installed and set up -- last step of the testing is to build the
   generated SRPM from a source-git repo using `mock --rebuild -r centos-stream-x86_64`).
+
+Once prereqs are met, you can run the tests like this:
+
+```
+$ pytest-3 tests/test_convert.py::test_conversions
+```
+
+It's also possible to invoke a single case (package), the example below also
+shows how to change the container engine (don't forget to build the image
+properly `CONTAINER_ENGINE='sudo podman' make build`):
```

Comment:
Why is building with `sudo podman` more "proper" than building with rootless?

If it's more proper, shouldn't this be the default?

Performance is mentioned above as a reason, but that seems to be a sloppy explanation. Comparing build times doesn't seem to reveal this: 

```
time make build
.
.
.
real    3m7.163s
user    2m4.944s
sys     0m36.025s
```

VS

```
time CONTAINER_ENGINE="sudo podman" make build
.
.
.
real    3m1.497s
user    2m16.163s
sys     0m37.762
```

(both builds ran with no images cached locally)

Reply:
https://github.com/packit/dist-git-to-source-git/blob/master/tests/test_convert.py#L73

I'll update the docs.

Reply:
I'm glad they finally fixed rootless containers performance, I haven't been using them for year+ because of how I/O was shredding my laptop.